### DISCUSSION TITLE
Starting the server through adb command

### DISCUSSION THREAD

**Discussion by paloss** [bk138/droidVNC-NG#247] [Q&A] [answered]
Hi,

I've been playing with the app and adb and i managed to set all the rights automatically through a script. The only thing i didn't succeed in is starting the server after installation. I still have to hit the "start" button manually. 

As a work around i'm now ending my script with adb reboot. When the devices boots the server starts automatic. 

I've been using the following line to try and start the server. I'm not getting any errors. Any pointers on what i might be doing wrong?

adb shell am start-foreground-service -n net.christianbeier.droidvnc_ng/.MainService -a net.christianbeier.droidvnc_ng.ACTION_START

**Comment by bk138** [comment]
It might be that opening the launcher activity once is required on some Android versions, but I don't remember exactly. You can try looking at logcat what the system says...

**Comment by bk138** [comment, marked as answer]
Generally, it's https://github.com/bk138/droidVNC-NG?tab=readme-ov-file#remote-control-via-the-intent-interface
